src/hw1/StringManipulator.java

```diff
@@ -1,24 +0,0 @@
-package hw1;
```

Comment:
Not sure why this file was deleted. The idea was to have you edit this file so that you may use the comments as a guide.